src/client/client.js

```diff
@@ -5,11 +5,9 @@ import ActionHTTPTransport from 'universal/utils/ActionHTTPTransport';
 import makeStore from './makeStore';
 import Root from './Root';
 
-const {routing} = window.__INITIAL_STATE__; // eslint-disable-line no-underscore-dangle
+// const {routing} = window.__INITIAL_STATE__; // eslint-disable-line no-underscore-dangle
```

Comment:
I'd say leave for now. If we remove it, we should also remove this part from the SSR since we no longer need to stringify & send it down from the server.